bundle/edu.gemini.phase2.skeleton.servlet/src/main/scala/edu/gemini/phase2/skeleton/factory/SpProgramFactory.scala

```diff
@@ -70,13 +70,13 @@ object SpProgramFactory {
     prog.setThesis(isThesis(proposal))
 
     prog.setPIInfo(piInfo(proposal))
-    hostNgoEmail(proposal) foreach { e => prog.setPrimaryContactEmail(e) }
+    hostNgoEmail(proposal) foreach prog.setPrimaryContactEmail
 
     // Note: not a typo -- "contact person" is an email
     gemEmail(proposal) foreach { e => prog.setContactPerson(e) }
 
-    minBand3Time(proposal) foreach { tv => prog.setMinimumTime(tv) }
-    timeAcctAllocation(proposal) foreach { alloc => prog.setTimeAcctAllocation(alloc) }
+    minBand3Time(proposal) foreach prog.setMinimumTime
+    timeAcctAllocation(proposal) foreach prog.setTimeAcctAllocation
```

Comment:
Right, I'm not sure if we should enforce it, but my preference if we do is to do validations early when the program is parsed so we can return a proper HTTP error code but maybe that would be too complicated